subsys/net/lib/nrf_provisioning/src/nrf_provisioning.c

```diff
@@ -167,7 +167,7 @@ static int nrf_provisioning_set(const char *key, size_t len_rd,
 	int len;
 	int key_len;
 	const char *next;
-	char time_str[sizeof(STRINGIFY(9223372036854775807))] = {0};
+	char time_str[sizeof(STRINGIFY(2147483647))] = {0};
```

Comment:
I think this is not right.. 
In Zephyr, `time_t` is 64bit, not 32bit.

Minimal LibC:
https://github.com/zephyrproject-rtos/zephyr/blob/main/lib/libc/minimal/include/sys/_types.h#L15

Picolib:
https://github.com/zephyrproject-rtos/picolibc/blob/main/newlib/libc/include/sys/_types.h#L234-L239

So leave the buffer size as it is. Otherwise you hide a Y2K38 problem here.

Reply:
In this case, time_str is used for provisioning interval in seconds not actual timestamp. So, there shouldn't be Y2K38 problem but I can revert the change.